ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp

```diff
@@ -72,8 +72,6 @@ LRESULT CALLBACK HookWndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
     GetWindowThreadProcessId(enigma::hWnd, &dwProcessId);
     AllowSetForegroundWindow(dwProcessId);
     SetForegroundWindow(enigma::hWnd);
-    SetActiveWindow(enigma::hWnd);
-    SetFocus(enigma::hWnd);
```

Comment:
Also, `SwitchToThisWindow` is a function.
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-switchtothiswindow

Reply:
> [This function is not intended for general use. It may be altered or unavailable in subsequent versions of Windows.]

Reply:
Sure, I was just pointing it out, I didn't mean to use it unless it was sensible.

Reply:
The advantage of using SetForegroundWindow over SetActiveWindow is with the former you can do it with any application process you know the window handle of, where as the latter can only be used on the current application process. I wanted to expand this at some point to add a function where you can choose a any window from any application to allow drag and drop for, and not just engima's main window. I don't know if i really want to do this because idk how useful it will be, but it is something I'd like to at least make an option down the line.